src/content_scripts/github/Github.ts

```diff
@@ -151,6 +151,12 @@ export default class Github {
         let currentProps = [];
         let href = (<HTMLAnchorElement>issue.querySelector('.js-navigation-open')).href;
         currentProps['platformId'] = Github.getPlatformIdFromUrl(href);
```

Comment:
update function to
```
    private static initOverviewItem(issue: HTMLElement) {
        const href = (<HTMLAnchorElement>issue.querySelector('.js-navigation-open')).href;
        let currentProps = [];
        currentProps['platformId'] = Github.getPlatformIdFromUrl(href);

        return VueInitializer.createComponent(
            `li#${issue.id} .mt-1.text-small.text-gray`,
            'fnd-meta-issue-funds', GithubOverviewItemFunds, currentProps, true);
    }
```
This we always use the same container. No need for creating to full GithubOverviewItemFunds components per issue.

Reply:
done :)